main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/rs/zerolog"
+	"github.com/statechannels/go-nitro/client"
+	"github.com/statechannels/go-nitro/client/engine"
+	"github.com/statechannels/go-nitro/client/engine/chainservice"
+	p2pms "github.com/statechannels/go-nitro/client/engine/messageservice/p2p-message-service"
+	"github.com/statechannels/go-nitro/client/engine/store"
+	"github.com/statechannels/go-nitro/rpc"
+	"github.com/statechannels/go-nitro/rpc/transport/ws"
+)
+
+func main() {
+	var pkString string
+	var msgPort, rpcPort int
+
+	flag.StringVar(&pkString, "pk", "2d999770f7b5d49b694080f987b82bbc9fc9ac2b4dcc10b0f8aba7d700f69c6d", "Specifies the private key for the client. Default is Alice's private key.")
+	flag.IntVar(&msgPort, "msgPort", 3005, "Specifies the tcp port for the  message service.")
+	flag.IntVar(&rpcPort, "rpcPort", 4005, "Specifies the tcp port for the rpc server.")
+	flag.Parse()
+	pk := common.Hex2Bytes(pkString)
+
+	logDestination := os.Stdout
+
+	storeA := store.NewMemStore(pk)
+	chain := chainservice.NewMockChain()
+	chainService := chainservice.NewMockChainService(chain, *storeA.GetAddress())
+	messageservice := p2pms.NewMessageService("127.0.0.1", msgPort, *storeA.GetAddress(), pk)
```

Comment:
Currently we never [add any peers](https://github.com/statechannels/go-nitro/blob/c67ab800fc9ff45151162a388378cddb1802d54a/client/engine/messageservice/p2p-message-service/service.go#L52)so the message service won't be able to communicate with anyone

Reply:
This [inspired me to make use of libp2p peer discovery](https://github.com/statechannels/go-nitro/pull/1207) so we don't have to worry about manually adding peers.